packages/ui/src/components/List/HeaderCell.tsx

```diff
@@ -56,6 +57,7 @@ type HeaderCellProps = {
   isOrdered?: boolean
   orderDirection?: 'asc' | 'desc' | 'none'
   onOrder?: (newOrder: 'asc' | 'desc') => void
+  tooltipInfo?: string
```

Comment:
Can we keep `tooltip` instead? That's the prop name we use for other components

Reply:
info sound better IMHO

Reply:
Let's go with `info` then 😄